### PR TITLE
Clarify source of indentation in docs (fixes #902)

### LINKED ISSUE #902: Indent by 4 instead of 2 spaces
Is 4 space indention currently hard-coded?

I (now) prefer 2 instead of 4 space indendation in XML files.

It's also recommended e.g. on https://maven.apache.org/guides/development/guide-documentation-style.html for `pom.xml`.

I tried to look for a configuration to change this on https://github.com/redhat-developer/vscode-xml/blob/main/docs/Formatting.md, but couldn't find any....

### PR DESCRIPTION
**Pull request by vorburger**
fixes #902

### CODE REVIEW

**Comment by vorburger** [comment]
@angelozerr ciao salut

File: docs/Formatting.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ## Formatting
 
+### Indentation
+
+The standard `editor.insertSpaces` & `editor.tabSize` [format settings](https://code.visualstudio.com/docs/editor/codebasics#_indentation) are used for [Spaces vs Tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI), and how many. To indent XML differently than e.g. Java, configure these settings language specifically ([example](https://github.com/vorburger/vorburger-dotfiles-bin-etc/blob/31c1953e550a21133ae10e017bb18fe829bc3711/dotfiles/code/settings.json#L27)).
```
Comment (fbricon) [inline docs/Formatting.md:5]: please embed the example in this file. This doc file can be viewed in VS Code directly, in offline mode.

Comment (vorburger) [reply docs/Formatting.md]: Done!

File: docs/Formatting.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
 ## Formatting
 
+### Indentation
+
+The standard `editor.insertSpaces` & `editor.tabSize` [format settings](https://code.visualstudio.com/docs/editor/codebasics#_indentation) are used for [Spaces vs Tabs](https://www.youtube.com/watch?v=SsoOG6ZeyUI), and how many. To indent XML differently than e.g. Java, configure language specific settings, by adding this to your `settings.json`:
```
[the remark this answers is not in the document]
Comment (fbricon) [reply docs/Formatting.md:5]: yeah, as much as I love Silicon Valley, we can do without the youtube link.

**Pull request merged**